Bug fix tank insulation nomenclature

ssankalp26 opened the pull request:
1. Fixed Nomenclature issue with the tank insulation materials
2. Created a reuse geometry flag for in the geometry proprocess to reduce the compute time. 
3. Fixed a bug in the mass_properties preprocess so that it now uses the vehicle from the geometry following the sequence

ssankalp26 committed:
Rename and update vacuum insulation material classes

Renamed Vacuum_Multilayer_Insulation to Vacuum_Cellular_Multilayer_Insulation and Vacuum_Gap_Multilayer_Insulation to Vacuum_Jacketed_Multilayer_Insulation. Updated all relevant imports and references in the materials library, vehicle setup, and coverage test to use the new class names for consistency and clarity.

ssankalp26 committed:
Merge remote-tracking branch 'origin/feature-vtk_visualization' into bug_fix-tank_insulation_nomenclature

ssankalp26 committed:
Update VERSION

ssankalp26 committed:
Merge remote-tracking branch 'origin/feature-vtk_visualization' into bug_fix-tank_insulation_nomenclature

ssankalp26 committed:
Update insulation material in fuel tank tests

Replaced Vacuum_Gap_Multilayer_Insulation with Vacuum_Cellular_Multilayer_Insulation in fuel tank volume tests. Added specific_density attribute to Vacuum_Cellular_Multilayer_Insulation material class for consistency.

ssankalp26 committed:
Merge branch 'develop' into bug_fix-tank_insulation_nomenclature

ssankalp26 committed:
code works for all segments

ssankalp26 committed:
Fix geometry and mass property copying logic

Improves deep copying of vehicle geometry and mass properties between mission segments to ensure control surface deflections are preserved. Adds print statement for fuel mass in compute_fuel_volume for debugging.

ssankalp26 committed:
Update fuel tank tests and plot configuration

Adjusted true fuel volume value and increased control points in fuel tank volume test. Simplified plot test by removing several fuel tank definitions and updated insulation material to Vacuum Jacketed Multilayer Insulation.

ssankalp26 committed:
got rid of stray print

ARmolloy committed:
Update Tutorial_07_lift_cruise_evtol_aircraft.ipynb

Reducing run time

ARmolloy committed:
Update mass_properties.py

ssankalp26 committed:
Update turbofan_network_test.py

ARmolloy committed:
Test Changes

ssankalp26 committed:
Update mass properties assignments in mission analysis

Changed mass properties assignments from 'aerodynamics' to 'geometry' in payload range diagram computation. Added support for copying vehicle weights to 'energy' analysis in mass properties preprocessing.

ssankalp26 committed:
Update compute_payload_range_diagram.py

ssankalp26 committed:
Update weights classes and geometry processing

Removed PRSEUS and aircraft_type settings from Conventional weights class and added PRSEUS to Conventional_BWB.

**Geometry pre-processing now synchronizes landing gear extension state. **
Updated Boeing_787 to use Conventional_Transport weights class for improved fidelity and composite settings.

mclarke2 commented on RCAIDE/Library/Attributes/Materials/Vacuum_Cellular_Multilayer_Insulation.py:
```diff
@@ -9,11 +9,11 @@
 
 #-------------------------------------------------------------------------------
-# Vacuum Multilayer Insulation
+# Vacuum Cellular Cellular Multilayer Insulation
```
Correct name

ssankalp26 replied:
Done

mclarke2 commented on VnV/Vehicles/Boeing_787.py:
```diff
@@ -920,7 +920,7 @@ def base_analysis(vehicle):
 
     # ------------------------------------------------------------------
     #  Weights
-    weights = RCAIDE.Framework.Analyses.Weights.Conventional()
```
Do we need to update the B737 and E190 aircrafts

ssankalp26 replied:
They do not have weight analysis in thte regressions

ssankalp26 replied:
DOne

ssankalp26 committed:
Update payload_range_test.py

ssankalp26 committed:
Update test truth values and geometry settings

Adjusted truth values in battery, mission segment, and turbofan tests for improved accuracy. Set unique_geometry to True in geometry analyses for multiple tests. Updated weights analysis to use Conventional_Transport in loading and trim diagram test. Minor docstring and solver type cleanup.

ssankalp26 committed:
Update Geometry.py

ssankalp26 committed:
Update battery_cell.py

ssankalp26 committed:
patch coverage improvement

ARmolloy reviewed (approved):
Looks all good! Discussed adding tank insulation weight to total aircraft weight in the future
